test/benchmark/api_v1_1/poller_tests.py

```diff
@@ -39,4 +39,9 @@ def test_runtime(self):
         assert_equal(True, ansible_ctl.collect_data(), message='Failed to collect footprint data!')
         self.__case_recorder.write_end()
 
-        parser.parse(self.__data_path)
+        LOG.info('Parse log and generate html reports')
+        try:
+            parser.parse(self.__data_path)
+        except Exception as err:
```

Comment:
Is this only to catch `RuntimeError` ? If so , just put `except RuntimeError as err` here to avoid swallow any unknown exceptions